ds/queue.py

```diff
@@ -0,0 +1,74 @@
+""" You are an owner of a hot dog stand (vegetarian hot dogs for Blaze and GameBot). """
+""" All of a sudden aliens take over the planet and capture the entire planet. They """
+""" are about to destroy your hot dog stand when they notice how good your (vegetarian)"""
+""" hot dogs are. Because of this, they make a bet with you: Simulate your hotdog stand """
+""" business using python code, and if you pass, humanity will be saved. Because you're """
+""" an overachiever (nerd), you want to implement this not only by modifying your old """
+""" stack code but ALSO USING AN IMMUTEABLE ARRAY OF SIZE 10. Good luck, the fate of """
+""" humanity lies in your hands."""
+
+class PersonStack:
+
+    def __init__(self,name):
+        self.name = name
+
+class StoreLineStack:
+
+    def __init__(self):
+
+        """ implement your stack here """
+
+        pass
+
+    def enqueue(self, new_name):
+
+        """ Rewrite one of your stack functions to work here """
+
+        pass
+
+
+    def dequeue(self):

```

Comment:
When we dequeue, we return the generic type, right? 

def dequeue(self) -> T:
    pass


Look at the other assignments for help on this